data-prepper-core/build.gradle

```diff
@@ -20,13 +20,13 @@ dependencies {
     implementation 'com.fasterxml.jackson.core:jackson-databind'
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
     implementation "javax.validation:validation-api:2.0.1.Final"
-    implementation "org.apache.bval:bval-extras:2.0.5"
-    implementation "org.apache.bval:bval-jsr:2.0.5"
     implementation "org.reflections:reflections:0.10.2"
     implementation 'io.micrometer:micrometer-core'
     implementation 'io.micrometer:micrometer-registry-prometheus'
     implementation 'io.micrometer:micrometer-registry-cloudwatch2'
     implementation 'software.amazon.awssdk:cloudwatch'
+    implementation 'org.hibernate.validator:hibernate-validator:7.0.2.Final'
+    implementation 'org.glassfish:jakarta.el:4.0.1'
```

Comment:
Just curious since we were talking about these dependencies recently. Originally you had thought that the Glassfish dependency was unnecessary because we just needed the built in JSR-303 annotations. What was your finding exactly that showed a need for the glassfish dependency?